docs/reference/edot-collector/troubleshooting/collector-oomkilled.md

```diff
@@ -0,0 +1,109 @@
+---
+navigation_title: EDOT Collector is out of memory
+description: Diagnose and resolve out-of-memory issues in the EDOT Collector using Go’s Performance Profiler.
+applies_to:
+  stack:
+  serverless:
+    observability:
+products:
+  - id: cloud-serverless
+  - id: observability
+  - id: edot-collector
+---
+
+# EDOT Collector is out of memory
+
+If your EDOT Collector is being terminated with an `OOMKilled` status, this usually indicates a memory leak or sustained memory pressure. You can use the Performance Profiler (`pprof`) tool to collect and analyze memory profiles, helping you identify the root cause.
+
+## Symptoms
+
+- EDOT Collector pod restarts with an `OOMKilled` status in Kubernetes.
+- Memory usage steadily increases before the crash.
+- The collector's logs don't show clear errors before termination.
+
+## Resolution
+
+Enable runtime profiling using the `pprof` extension and then gather memory heap profiles from the affected pod:
```

Comment:
```suggestion
Turn on runtime profiling using the `pprof` extension and then gather memory heap profiles from the affected pod:
```

We prefer "turn on" or "activate" to avoid ableist language